source/Calamari.Shared/Commands/Support/ICommandWithArguments.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using Calamari.Deployment.Conventions;
+
+namespace Calamari.Commands.Support
+{
+    /// <summary>
+    /// A command that requires the command line arguments passed to it. We are transitioning away from this interface  
+    /// </summary>
+    public interface ICommandWithArguments
+    {
+        int Execute(string[] commandLineArguments);
+    }
+
+    public interface ICommand
```

Comment:
Could we move this back out to its own file so it shows as a diff rather than a delete of `ICommand.cs`?